toolchest_client/api/query.py

```diff
@@ -38,55 +39,53 @@ class Query:
     """
 
     # Period (in seconds) between requests when waiting for job(s) to finish executing.
-    WAIT_FOR_JOB_DELAY = 1
-    # Multiple of seconds used when pretty printing job status to output.
-    PRINTED_TIME_INTERVAL = 5
+    WAIT_FOR_JOB_DELAY = 5
```

Comment:
Updated the delay to 5 seconds for the API call, as we were already waiting 5 seconds for the printed interval. This leads to someone off-kilter status updates, as they're not updated at the five second mark.